### DISCUSSION TITLE
Decode

### DISCUSSION THREAD

**Discussion by alfirus** [mebjas/html5-qrcode#167] [Q&A]
Hi everyone.

I managed to up this script and it can detect QR code. Now I wonder how to make it decode the QR Code.

**Comment by mebjas** [comment]
If it detects the QR Code, it should be returning a decoded QR Code via callback as well. Can you share your code snippet?

**Comment by Waltz123** [comment]
I too have the same question as Alfirus. I see that there's a callback once the code has been scanned. The problem is that there's nothing in the documentation for "Pro Mode" that shows how to refer to the scanned code inside the callback function. For example:

const qrCodeSuccessCallback = message => {
   console.log("The code has been successfully scanned");
   console.log(message); // This doesn't work
   console.log(html5QrCode); // This has information, but not the actual text string that was scanned
}

So to elaborate on Alfrius' question: What's the name of the variable or object inside the callback function that has the actual text string that was contained in the scan?

Thank you so much for your time!

Regards,
Jonathan

**Reply by alfirus** [reply]
Thanks for elaborate my question. Currently i'm in another project and can't figure it out yet.

**Comment by ROBERT-MCDOWELL** [comment]
maybe you should check the sources code to know that
